app/assets/stylesheets/components/_header-notice.scss

```diff
@@ -21,11 +21,11 @@ $nhs_brand_color: #005EB8;
 
 .app-c-header-notice__title {
   color: govuk-colour("white");
-  margin: 0;
```

Comment:
This change doesn't do anything - except increase the sass compilation time by a tiny amount. I'd suggest keeping it as 0, it's more readable anyway.

Reply:
Good point